app/src/main/java/com/rage/clamber/Fragments/UserInfoFragment.java

```diff
@@ -71,12 +81,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         ButterKnife.bind(this, rootView);
 
         mainUser = getArguments().getParcelable(HomePage.ARG_USER);
-
+        if (mainUser != null) {
```

Comment:
Generally not bad to check for null. Mostly I was just trying to figure out if that was ever an expected case, I would leave it in just in case, though maybe log an error if it is null for whatever reason. That way if you make a mistake sometime in the future and allow it to be null you'll get a nice message about it. Hardly crucial though.